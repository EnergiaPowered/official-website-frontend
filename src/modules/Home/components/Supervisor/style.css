.sup-name {
  font-weight: 800;
}

.sup-univ {
  margin-bottom: 40px;
}

.sup-image {
  max-width: 100%;
  height: auto;
}

/* all paragraphs */
p {
  margin-bottom: 20px;
}
