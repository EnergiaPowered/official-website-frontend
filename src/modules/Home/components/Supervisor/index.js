import React from "react";

// css
import "./style.css";

// assets
import supervisor from "assets/supervisor.jpg";

export default () => {
  return (
    <section className="container bg-section component-font" id="supervisor">
      <h2 className="section-title text-center"> our supervisor </h2>

      <main className="row">
        <section className="col-md-6 col-sm-12">
          <p className="sup-name">DR. Samah Elshafiey</p>
          <p className="sup-univ"> Faculty of Engineering, Cairo University </p>

          <p className="sup-job">
            Assistant Professor, Engineering Mathematics and Physics Department
            Co-Director, Technical Center for Career Development{" "}
          </p>
        </section>
        <section className="col-md-6 col-sm-12">
          <img
            src={supervisor}
            alt="Doctor Samah, Our supervisor"
            className="sup-image"
          />
        </section>
      </main>
    </section>
  );
};
