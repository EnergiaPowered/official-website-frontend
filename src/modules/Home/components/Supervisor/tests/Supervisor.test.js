import React from 'react';
import Supervisor from '../index';

import { create } from "react-test-renderer";

// because it's static component and doesn't change often
it('matches snapshot', () => {
  const component = create(<Supervisor />);

  expect(component.toJSON()).toMatchSnapshot()
});
