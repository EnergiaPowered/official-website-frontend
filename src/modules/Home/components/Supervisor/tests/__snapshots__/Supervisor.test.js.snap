// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<section
  className="container bg-section component-font"
  id="supervisor"
>
  <h2
    className="section-title text-center"
  >
     our supervisor 
  </h2>
  <main
    className="row"
  >
    <section
      className="col-md-6 col-sm-12"
    >
      <p
        className="sup-name"
      >
        DR. Samah Elshafiey
      </p>
      <p
        className="sup-univ"
      >
         Faculty of Engineering, Cairo University 
      </p>
      <p
        className="sup-job"
      >
        Assistant Professor, Engineering Mathematics and Physics Department Co-Director, Technical Center for Career Development
         
      </p>
    </section>
    <section
      className="col-md-6 col-sm-12"
    >
      <img
        alt="Doctor Samah, Our supervisor"
        className="sup-image"
        src="supervisor.jpg"
      />
    </section>
  </main>
</section>
`;
